### PR TITLE
Issue with swiftmailer failing to connect

### PR DESCRIPTION
**Pull request by TNAJanssen**
https://github.com/swiftmailer/swiftmailer/issues/490

### CODE REVIEW

**Comment by matthiasnoback** [comment]
Thanks @TNAJanssen for submitting this PR.

@cmodijk Could you take a look as well please?

**Comment by cmodijk** [comment]
Why should we start the connection? The `flushQueue` is already doing that if there is a message to send right? 
- https://github.com/swiftmailer/swiftmailer/blob/5.x/lib/classes/Swift/MemorySpool.php#L73-L75

I think the part about stopping the transport should stay because of possible timeouts.

File: src/Plugin/SwiftMailerPlugin/ClearSpools.php [hunk 1]
```diff
@@ -39,5 +42,8 @@ public function cleanUp()
                 $this->logger->error(sprintf('Exception occurred while flushing email queue: %s', $exception->getMessage()));
             }
         }
+        foreach ($this->transports as $transport) {
```
Comment (cmodijk) [inline src/Plugin/SwiftMailerPlugin/ClearSpools.php]: Why should we stop all the transports if we only use one of the transports? 
- https://github.com/LongRunning/LongRunning/blob/master/src/Plugin/SwiftMailerPlugin/ClearSpools.php#L37

**Comment by matthiasnoback** [comment]
Thanks @cmodijk. @TNAJanssen would you care to elaborate?

**Comment by cmodijk** [comment]
@TNAJanssen Do you have the tine to look at this?

**Comment by TNAJanssen** [comment]
@cmodijk i will check it out tomorrow!

**Comment by TNAJanssen** [comment]
@cmodijk you're right, i will try to fix this this week!

**Comment by cmodijk** [comment]
@TNAJanssen Do you have the time to fix this one or should i fix it?

**Pull request closed without merging**